torch/_dynamo/debug_utils.py

```diff
@@ -240,7 +240,7 @@ def save_graph_repro(fd, gm, args, compiler_name):
 def isolate_fails(fx_g, args, compiler_name: str, env=None):
     if env is None:
         env = {}
-    subdir = f"{minifier_dir()}/isolate"
+    subdir = os.path.join(os.getcwd(), "isolate")
```

Comment:
Note, os.getcwd() is kind of awkward for benchmark/dynamo/torchbench.py and friends, because these programs change their working directory after startup. It might be nice for those scripts to record their original working dir and tell minifier to dump there for user friendliness

Reply:
I think this is just used in minifier_launcher.py, but I can double check. This also does affect the debug_dir_root, I can have the benchmark suites set that somewhere intuitive.